style/themes/lcars.css

```diff
@@ -329,24 +336,32 @@ label {
 
 .form-group.has-error.login-box-msg::before {
   content: "ACCESS DENIED";
-  display: block;
   margin: -20px -20px 20px;
   padding: calc(45px + 0.7em) 0 0;
   position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 130px;
-  color: #d43;
-  font-size: 3.825em;
```

Comment:
Just a note:

`font-size: 3.825em;` value was here as a fallback for the calculation below.I have no idea if there are browsers today that don't support the combination of `calc()`, `vw` and `var()` (custom properties).

I didn't test this with all possible browser/OS/device combinations, but this value is probably unnecessary.